Modules/_multiprocessing/multiprocessing.c

```diff
@@ -180,17 +161,17 @@ PyInit__multiprocessing(void)
         else
             py_sem_value_max = PyLong_FromLong(SEM_VALUE_MAX);
         if (py_sem_value_max == NULL)
-            return NULL;
+            return -1;
         PyDict_SetItemString(_PyMp_SemLockType.tp_dict, "SEM_VALUE_MAX",
                              py_sem_value_max);
     }
     PyModule_AddObject(module, "SemLock", (PyObject*)&_PyMp_SemLockType);
 #endif
 
     /* Add configuration macros */
-    temp = PyDict_New();
+    PyObject *temp = PyDict_New();
     if (!temp)
-        return NULL;
+        return -1;
 
 #define ADD_FLAG(name)                                            \
```

Comment:
You must replace return NULL with return -1.

Reply:
@vstinner now I miss working in c++ and using nullptr instead of NULL to avoid this error